MAINTAINERS

```diff
@@ -0,0 +1,83 @@
+# PowerShell-Docker maintainers file
+# Adapted from Moby's maintainers file
+#
+# This file describes the maintainer groups within the PowerShell/PowerShell-Docker project.
+#
+# It is structured to be consumable by both humans and programs.
+# To extract its contents programmatically, use any TOML-compliant
+# parser.
```

Comment:
I'm fin with this being plain markdown if  you prefer.  I'm sure docker has a reason for using this format, but I don't

Reply:
I like it being TOML compliant.